rp2040-hal/src/gpio/dynpin.rs

```diff
@@ -305,7 +305,7 @@ impl DynPin {
     /// must be at most one corresponding [`DynPin`] in existence at any given
     /// time.  Violating this requirement is `unsafe`.
```

Comment:
Just a side note, as it wasn't changed in this PR: The comment "Violating this requirement is `unsafe`" is a little bit off. The function is `unsafe` because the caller must guarantee that `DynPin`s are unique. Violating such a condition is not `unsafe`, it's plain wrong. It may cause UB, or other bugs. But saying that violation the safety conditions of an `unsafe` function is `unsafe` doesn't convey any useful meaning.